uint16 JOYSTICK_MIN = 0
uint16 JOYSTICK_MID = 32767
uint16 JOYSTICK_MAX = 65535

uint16 TRIGGER_MIN = 0
uint16 TRIGGER_MID = 511
uint16 TRIGGER_MAX = 1023

bool a
bool b
bool x
bool y
bool left_joystick
uint16 left_joystick_x 
uint16 left_joystick_y
uint16 left_trigger
bool left_bumper
bool right_joystick
uint16 right_joystick_x
uint16 right_joystick_y
uint16 right_trigger 
bool right_bumper
bool up
bool down
bool left
bool right
bool start
bool select
bool home
